pkg/operator/ceph/cluster/osd/spec.go

```diff
@@ -305,12 +302,8 @@ func (c *Cluster) makeDeployment(osdProps osdProperties, osd OSDInfo) (*apps.Dep
 			},
 			Template: v1.PodTemplateSpec{
 				ObjectMeta: metav1.ObjectMeta{
-					Name: appName,
-					Labels: map[string]string{
-						k8sutil.AppAttr:     appName,
-						k8sutil.ClusterAttr: c.Namespace,
-						osdLabelKey:         fmt.Sprintf("%d", osd.ID),
-					},
+					Name:   appName,
+					Labels: c.getOSDLabels(osd.ID, failureDomainValue, strconv.FormatBool(osdProps.portable)),
```

Comment:
how about passing a bool and then format inside the getOSDLabels() method?

Reply:
done